### PR TITLE
init db schemas

### PR DESCRIPTION
**Pull request by developerKhusanjon**

### CODE REVIEW

File: .bsp/sbt.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.7.2","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Users/khusanjon/Library/Java/JavaVirtualMachines/openjdk-19/Contents/Home/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/khusanjon/Library/Application Support/JetBrains/IdeaIC2022.2/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=/Users/khusanjon/Library/Application%20Support/JetBrains/IdeaIC2022.2/plugins/Scala/launcher/sbt-launch.jar"]}
```
Comment (VladimirBaklan) [inline .bsp/sbt.json]: Please fix .gitignore

File: docker/sql/docmg.sql [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+-- Admins
+CREATE TABLE admins (
+  id uuid NOT NULL,
+  name character varying NOT NULL,
+  role character varying NOT NULL,
+  year_of_employment smallint NOT NULL,
+  company_id integer NOT NULL
+);
+
+ALTER TABLE admins
```
Comment (VladimirBaklan) [inline docker/sql/docmg.sql]: You should define constraints when you create a table without using `alter table`

File: docker/sql/docmg.sql [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ADD CONSTRAINT admins_id PRIMARY KEY (id);
+ALTER TABLE admins
+ADD FOREIGN KEY (company_id) REFERENCES companies (id);
+
+-- Employees
```
Comment (VladimirBaklan) [inline docker/sql/docmg.sql:12]: Comments are too obvious here, no need to add it

File: docker/sql/docmg.sql [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+ADD CONSTRAINT employees_id PRIMARY KEY (id);
+ALTER TABLE employees
+ADD FOREIGN KEY (company_id) REFERENCES companies (id);
+
+-- WebForms
+CREATE TABLE web_forms (
+  id serial NOT NULL,
+  PRIMARY KEY (id),
+  title character varying NOT NULL
+  content character varying NOT NULL
```
Comment (VladimirBaklan) [inline docker/sql/docmg.sql]: What is `content`?

Comment (VladimirBaklan) [reply docker/sql/docmg.sql]: And how would you store submissions?

File: docker/sql/docmg.sql [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+);
+
+ALTER TABLE admins
+ADD CONSTRAINT admins_id PRIMARY KEY (id);
+ALTER TABLE admins
+ADD FOREIGN KEY (company_id) REFERENCES companies (id);
+
+-- Employees
+CREATE TABLE employees (
+  id uuid NOT NULL,
```
Comment (VladimirBaklan) [inline docker/sql/docmg.sql]: It makes more sense to use auto-generated primary keys, like SERIAL/BIGSERIAL

File: docker/sql/docmg.sql [hunk 6]
```diff
@@ -0,0 +1,47 @@
+-- Database
+CREATE DATABASE docmg;
+\c docmg;
+
+-- Companies
+CREATE TABLE companies (
+  id serial NOT NULL,
+  PRIMARY KEY (id),
+  name character varying NOT NULL,
```
Comment (VladimirBaklan) [inline docker/sql/docmg.sql]: `character varying` => `varchar`

File: docker/sql/docmg.sql (hunk 2 above)
Comment (VladimirBaklan) [inline docker/sql/docmg.sql]: Why do you store users and admins separately?

File: project/plugin.sbt [hunk 7]
```diff
@@ -0,0 +1 @@
+addSbtPlugin("io.github.davidmweber" % "flyway-sbt" % "7.4.0")
```
Comment (VladimirBaklan) [inline project/plugin.sbt:1]: I don't see configurations for plugin

**Review by VladimirBaklan — CHANGES_REQUESTED** [review]

File: docker/sql/docmg.sql [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+  name varchar(100) NOT NULL,
+  role varchar(100) NOT NULL,
+  company_id integer REFERENCES companies
+);
+
+-- WebForms
+CREATE TABLE web_forms (
+  id serial PRIMARY KEY,
+  title varchar(100) NOT NULL
+  description varchar NOT NULL
```
Comment (VladimirBaklan) [inline docker/sql/docmg.sql]: What is the description?

Comment (developerKhusanjon) [reply docker/sql/docmg.sql]: I just added for extra info about webform... it describes what kind of form you are submitting

Comment (VladimirBaklan) [reply docker/sql/docmg.sql]: How would you store elements from webform? Webform has dynamic configuration as described in the task

Comment (developerKhusanjon) [reply docker/sql/docmg.sql]: I'm looking for this but for now, I haven't an exact solution... maybe I can add another table as form_field or like that with ManyToOne relationship with webform

Comment (developerKhusanjon) [reply docker/sql/docmg.sql]: Could do suggest something, please?

Comment (developerKhusanjon) [reply docker/sql/docmg.sql:24]: Could do suggest something, please?

File: src/main/scala/dev/khusanjon/docmngsys/company/CompanyRegistry.scala [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import dev.khusanjon.docmngsys.configuration.PostgresConfig
+import doobie.implicits._
+import doobie._
+
+import scala.collection.immutable
+
+//#company-case-classes
+final case class Company(name: String, address: String)
```
Comment (VladimirBaklan) [inline src/main/scala/dev/khusanjon/docmngsys/company/CompanyRegistry.scala:14]: Why final?

**Pull request closed without merging**